Resources/ProxyProvider/test/Test_ProxyProviderFactory.py

```diff
@@ -0,0 +1,61 @@
+__RCSID__ = "$Id$"
+
+import os
+import mock
+import unittest
+
+from DIRAC import S_OK, S_ERROR, gLogger
+from DIRAC.Resources.ProxyProvider.ProxyProviderFactory import ProxyProviderFactory
+
+
+thisPath = os.path.dirname(os.path.abspath(__file__)).split('/')
+rootPath = thisPath[:len(thisPath) - 3]
```

Comment:
```suggestion
rootPath = thisPath[:-3]
```
But actually
```python
from DIRAC import rootPath 
```
is what these two lines do?